packages/eslint-plugin/src/rules/prefer-readonly-parameter-types.ts

```diff
@@ -5,9 +5,11 @@ import * as util from '../util';
 
 type Options = [
   {
+    allow?: Array<util.TypeOrValueSpecifier>;
```

Comment:
Nit: we generally go with `[]` (I'm surprised this didn't fail on a lint complaint?)

```suggestion
    allow?: util.TypeOrValueSpecifier[];
```